python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -39,7 +39,7 @@ class LocalFile(IGangaFile):
                                      })
     _category = 'gangafiles'
     _name = "LocalFile"
-    _exportmethods = ["location", "remove", "accessURL", "processOutputWildcardMatches", "hasMatchedFiles"]
+    _exportmethods = ["location", "remove", "accessURL"]
```

Comment:
`processOutputWildcardMatches` was exclusively only used in the `Job.py` in favor of `setLocation` which wasn't defined for this class (and probably should have been).
I can put the `processOutputWildcardMatches` method back and add a warning that this shouldn't be used and will likely be removed soon. As for `hasMatchedFiles` I can add it back in, I just assumed this was only an internal thing but if people want access to it then there's no particular problem that I can see.